.github/workflows/component.yml

```diff
@@ -61,10 +61,18 @@ jobs:
           persist-credentials: false
           ref: ${{ inputs.ref || '' }}
 
+      - name: Read component config
```

Comment:
Wouldn't it be easier to create 2 global vars/envs with list of components and then use [contains](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#contains) to check if it in the string/list? Less code and same readability.